single-gs-factsheet.php

```diff
@@ -0,0 +1,95 @@
+<?php get_header(); ?>
+
+	<main class="degree-program-single">
+		<?php get_template_part( 'parts/headers' ); ?>
+
+		<?php if ( have_posts() ) : while ( have_posts() ) : the_post(); ?>
+
+			<header>
+				<hgroup>
+					<h1><?php the_title(); ?></h1>
+				</hgroup>
+			</header>
+
+			<section class="row single gutter pad-ends">
+
+				<div class="column one">
+					<?php $factsheet_data = WSUWP_Graduate_Degree_Programs()::get_factsheet_data( get_the_id() ); ?>
```

Comment:
I don't think we need the parentheses after the class name. (Edit: confirmed after testing locally :)